meta-oe/recipes-devtools/protobuf/protobuf_5.29.4.bb

```diff
@@ -52,6 +52,7 @@ TEST_SRC_DIR = "examples"
 LANG_SUPPORT = "cpp ${@bb.utils.contains('PACKAGECONFIG', 'python', 'python', '', d)}"
 
 LDFLAGS:append:riscv32 = " -latomic"
+CXXFLAGS:append:mipsarch = " -latomic"
```

Comment:
Lets use `mipsarcho32` since the issue mostly is with mips 32bit with Old ABI
I am wondering why it does not work when appended to LDFLAGS like we do for riscv32 for mips in same way

Reply:
Already I have tested all methods with (LDFLAGS) ... and I have got same error

> LDFLAGS:append:mipsarch = " -latomic"
and
 LDFLAGS:append:mips = " -latomic"
and 
LDFLAGS:append:mipsarcho32 = " -latomic"

Only with CXXFLAGS Solved the error !!
> CXXFLAGS:append:mipsarch = " -latomic"
or
CXXFLAGS:append:mipsarcho32 = " -latomic"

Reply:
Can you post verbose logs ? especially I would like to see the failing cmdline

Reply:
logs in attach and I have tested

> LDFLAGS:append:mipsarch = " -latomic"
[log.do_compile.txt](https://github.com/user-attachments/files/19412368/log.do_compile.txt)

> LDFLAGS:append:mipsarcho32 = " -latomic"
[log.do_compile.txt](https://github.com/user-attachments/files/19412347/log.do_compile.txt)